packages/framework-core/src/booster-event-dispatcher.ts

```
@@ -96,7 +98,12 @@ export class BoosterEventDispatcher {
           const eventInstance = createInstance(eventClass.class, eventEnvelope.value)
           const register = new Register(eventEnvelope.requestID, eventEnvelope.currentUser)
           logger.debug('Calling "handle" method on event handler: ', eventHandler)
-          await eventHandler.handle(eventInstance, register)
+          try {
+            await eventHandler.handle(eventInstance, register)
+          } catch (e) {
+            const globalErrorDispatcher = new BoosterGlobalErrorDispatcher(config, logger)
+            throw await globalErrorDispatcher.dispatch(new EventHandlerGlobalError(eventInstance, e))
```

Same here, should we allow ignoring an error?